src/components/paradox/results/personality/career/inline-jobs/job.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import {faLocationDot} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
+import DangerousHTML from "lib/helpers/dangerous-html";
+import style from "./style.scss";
+
+function InlineJob({job, jobSource, jobSourceURL, translate}) {
+  if(!job || Object.keys(job).length === 0) {
+    return (
+      <div className={`${style.inlineJob} ${style.empty}`}>
+        <div>
+          <DangerousHTML html={translate("no_jobs", {job_source: jobSource})} />
+        </div>
+        <div>
+          <a
+            className={style.discoverJobsButton}
+            href={jobSourceURL[jobSource]}
+          >
+            {translate("discover_jobs")}
+          </a>
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div className={style.inlineJob}>
+      <div>
+        <div className={style.title}>{job.title}</div>
+        {job.company && <div className={style.company}>{job.company}</div>}
+        {job.location && (
+          <div className={style.location}>
+            <Icon className={style.jobIcon} icon={faLocationDot} />
+            {job.location}
+          </div>
+        )}
+      </div>
+      <div>
+        <a className={style.applyNowButton} href={job.url}>
```

Comment:
Is this required? Might need a conditional around it

Reply:
OK Added conditionals! I might want to just change the button to disabled later.